charts/minecraft/values.yaml

```diff
@@ -162,6 +162,8 @@ minecraftServer:
   serviceType: ClusterIP
   ## Set the port used if the serviceType is NodePort
   # nodePort:
+  ## Set the external port of the service, usefull when using the LoadBalancer service type
+  # servicePort:
```

Comment:
Even though there's also the default on the variable usage, I'm wondering pre-declaring the default here also would help self-document it.

Reply:
Agreed, since it also makes a difference with the default service type: `ClusterIP`. I'll change it.